src/main/resources/default/assets/tycho/scripts/charts.js.pasta

```diff
@@ -71,7 +71,7 @@ function inlineChart(selector, data, color, type) {
 }
 
 function _inlineBarChart(selector, data, color, roundedBars) {
-    let ctx = document.querySelector(selector);
+    let ctx = sirius.querySelector(selector);
```

Comment:
Should we apply our JS codestyle / naming scheme to this file? I.e. prefixing viarable names that reference a DOM element with _?